### PR TITLE
Enhancement: Update Quickstart Guide

### PR DESCRIPTION
**Pull request by Brijeshthummar02**
issue #373 

i've added QuickStart.md file with its reference in main readme.md file.

### CODE REVIEW

**Comment by Brijeshthummar02** [comment]
@phyleria please go through it.

**Comment by Brijeshthummar02** [comment]
@phyleria any update?

**Comment by Brijeshthummar02** [comment]
@phyleria have a look at pr.

File: submissions/Quickstart_Guide.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+chimoney = Chimoney(api_key='YOUR_API_KEY')
+
+response = chimoney.payout(amount=100, currency='USD', recipient='recipient@example.com')
+
+print('Payout successful:', response)
+```
+
+## Step 4: Overview of Key Features
+
+The Chimoney API provides several key functionalities, including:
```
Comment (phyleria) [inline submissions/Quickstart_Guide.md:72]: Amazing article @Brijeshthummar02!

Just a few changes:

- Please remove the last 2 on this list i.e `Balance retrieval` and `Transaction History` and replace them with: 
  - Multi-currency wallets
  - Payment Requests
  - Redemption of value
Also, add detailed descriptions for each functionality.

For example for `Payouts` a sample well-detailed description can be: 

> With Chimoney’s API, you can initiate seamless payouts to over 130 countries. Whether it's to banks, mobile money, airtime, or even Interledger-enabled accounts.

File: submissions/Quickstart_Guide.md [hunk 2]
```diff
@@ -71,7 +71,7 @@ print('Payout successful:', response)
 
 The Chimoney API provides several key functionalities, including:
 
-- **Payouts**: Send funds to users quickly and easily.
+- **Payouts**: With Chimoney’s API, you can initiate seamless payouts to over 130 countries via bank, mobile money, airtime, or even Interledger-enabled accounts.
 - **Multi-Currency Wallets**: Manage balances in multiple currencies, enabling smooth cross-border payments and better control over currency exchanges.
 - **Payment Requests**: Request payments from customers or clients, making invoicing and collections easier with a streamlined API integration.
 - **Redemption of Value**: Offer users the ability to redeem Chimoney value into services or products, giving them flexibility on how they use their balance.
```
Comment (phyleria) [inline submissions/Quickstart_Guide.md:77]: Please rewrite this `- **Redemption of Value**: Offer users the ability to redeem Chimoney value into services or products, giving them flexibility on how they use their balance.`

to

'- **Redemption of Value**: Offer users the ability to redeem Chimoney through options such as bank transfers, Mobile Money, gift cards, or airtime.`

Thanks.

**Comment by Brijeshthummar02** [comment]
@phyleria is it good to go?

**Review by phyleria — APPROVED** [review]
LGTM 🎉

**Comment by Brijeshthummar02** [comment]
@phyleria where do i've to claim the $25 bounty?

**Comment by phyleria** [comment]
The reward, will be sent to your email before the end of hacktoberfest. You will be notified when it is. Thanks!

**Comment by Brijeshthummar02** [comment]
alright, as it is approved feel free to merge it anytime.

**Comment by Brijeshthummar02** [comment]
@phyleria i haven't received any mail regarding rewards claim that you were saying.

**Comment by phyleria** [comment]
Hi @Brijeshthummar02 

We haven't sent out rewards yet. You'll receive a notification when we do. Thanks for your patience.

**Comment by Brijeshthummar02** [comment]
@phyleria it is almost 3 weeks from it do i'll get bounty?
appreciate if you send it as soon as possible.

**Comment by phyleria** [comment]
Hi @Brijeshthummar02 

Please join the discord Community [here](https://discord.gg/FAZZpZCZ). I have been sharing updates there. Thank you

**Pull request merged**